code/public/index.html

```diff
@@ -13,13 +16,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
-  </head>
+  <title>Technigo React App</title>
```

Comment:
Don't forget to change this little thingy to reflect your project ;) 
No biggie, but still a nice thing to have!